css/style.css

```diff
@@ -0,0 +1,73 @@
+body{
+	padding: 40px 20px;
+		font-family: sans-serif;
+	font-size: 18px;
+	border-top: 8px solid #79A6EA;
+	margin-top: 10px;
+	    color: #6C6C6C;
+} 
+
+
+body{
+	background: #79A6EA; /* For browsers that do not support gradients */
+    background: -webkit-linear-gradient(white, #79A6EA); /* For Safari 5.1 to 6.0 */
```

Comment:
@Uliana202 I see your using vendor prefixes here. Would you be able to explain what they're used for? The code looks fine.